### ISSUE TITLE
Filter function - Accessibility compliance 

### ISSUE THREAD

**Issue by tatianazt** [pods-framework/pods#3686]
Include in the parameters of filters function one option for the label of each field in order to comply to WCAG2 (web content accessibility guide) 1.3.1 - Associate labels properly with their controls. See http://www.w3.org/TR/WCAG20-TECHS/F68.html. 

If an user doesn´t want to use it, he could pass null for the corresponding field in the array.

In the same way, the search input should also accept a label. 

Besides, there should be also one more parameter to ask if the user wants to show the search input (sometimes just the select is enough).
